### PR TITLE
update button and link variants; colors

### PR DESCRIPTION
**Pull request by braxex**
## Description

Our button and link variants needed to be updated so we could fully align with CMSDS styles.

Note: This looks like a lot, but it's mostly just styling updates to line up with the real work, which is mostly contained in `theme.ts`.

### How to test

1. Click through and test that buttons and links all look fabulous. 

## Code author checklist
- [x] I have performed a self-review of my code
- [x] ~~I have added [thorough](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research) tests~~
- [x] ~~I have added analytics, if necessary~~
- [x] ~~I have updated the documentation, if necessary~~

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/styles/theme.ts [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+export const svgFilters = {
+  primary:
+    "brightness(0) saturate(100%) invert(30%) sepia(93%) saturate(1282%) hue-rotate(181deg) brightness(91%) contrast(101%)",
+  primary_darker:
+    "brightness(0) saturate(100%) invert(19%) sepia(43%) saturate(3547%) hue-rotate(185deg) brightness(97%) contrast(101%)",
+  white:
+    "brightness(0) saturate(100%) invert(100%) sepia(0%) saturate(7500%) hue-rotate(142deg) brightness(115%) contrast(115%);",
+  gray_lighter:
+    "brightness(0) saturate(100%) invert(91%) sepia(0%) saturate(89%) hue-rotate(162deg) brightness(97%) contrast(93%);",
+};
```
Comment (gmrabian) [inline services/ui-src/src/styles/theme.ts:13]: 🤯 I don't know what kinda brain blast stuff this is but it's working

File: services/ui-src/src/components/fields/DynamicField.tsx [hunk 2]
```diff
@@ -80,17 +82,14 @@ const sx = {
...
     height: "1.25rem",
+    _hover: {
+      filter: svgFilters.primary_darker,
+    },
   },
   appendButton: {
-    minWidth: "202px",
-    minHeight: "42px",
+    width: "202px",
+    height: "42px",
```
Comment (gmrabian) [inline services/ui-src/src/components/fields/DynamicField.tsx:91]: Could we rem it?

**Review by gmrabian — APPROVED** [review]
Looks good! I didn't see any issues after clicking around

File: services/ui-src/src/components/fields/DynamicField.tsx (hunk 2 above)
Comment (braxex) [reply services/ui-src/src/components/fields/DynamicField.tsx]: Good call.

**Pull request merged**